Resources/public/js/views/services/plugins/extensions/ez-draftserversidevalidation.js

```diff
@@ -32,20 +32,24 @@ YUI.add('ez-draftserversidevalidation', function (Y) {
          */
         _parseServerFieldsErrors: function (response, serverSideErrorCallback) {
             var serverSideFieldsError = [],
-                error = response.document.ErrorMessage,
-                fieldsError = error.errorDetails ?  error.errorDetails.fields : [];
+                error,
+                fieldsError;
 
             if (serverSideErrorCallback) {
-                fieldsError.forEach(function (field) {
-                    field.errors.forEach(function (error) {
-                        var serverSideFieldErrorDetails;
+                if (response.document) {
```

Comment:
it's a bit weird to check the response after the callback, why not write it like:
```js
var serverSideFieldsError = [],
    error,
    fieldsError;

if ( !response.document ) {
    return;
}
error = response.document.ErrorMessage;
fieldsError = error.errorDetails ?  error.errorDetails.fields : [];
// ...
```